### ISSUE TITLE
Add favorite places

### ISSUE THREAD

**Issue by ilpianista** [dcaliste/maep-qt#21]
Hi, it would be nice if I could mark a place as favorite for quicker access.

**Comment by dcaliste** [comment]
You're right. That's a good suggestion. I'll try to implement this in the coming weeks.
